src/Operators/remapping.jl

```diff
@@ -54,60 +56,102 @@ function linear_remap_op(target::AbstractSpace, source::AbstractSpace)
     return W .* J
 end
 
+"""
+    overlap_weights(target::T, source::S) where {T <: SpectralElementSpace1D{<:GridTopology, Quadratures.GL{1}},
+            S <: SpectralElementSpace1D{<:GridTopology, Quadratures.GL{1}}}
+
+Computes local weights of the overlap mesh for `source_topo` to `target_topo` topologies.
+
+Returns sparse matrix `W` where entry `W_{ij}` gives the area that target element `i` overlaps source element `j`.
+"""
+function overlap_weights(
+    target::T,
+    source::S,
+) where {
+    T <: SpectralElementSpace1D{<:IntervalTopology, Quadratures.GL{1}},
+    S <: SpectralElementSpace1D{<:IntervalTopology, Quadratures.GL{1}},
+}
+    # Calculate element overlap pattern
+    # X_ov[i,j] = overlap length between target elem i and source elem j
+    X_ov = fv_overlap_1D(target, source; dims = :x)
```

Comment:
I think it would be more natural to use the Geometry point types  here rather than `dims` symbol.

Reply:
That's a good point. It's better to be consistent with the existing API

Reply:
Yes that's a nice idea!